### PR TITLE
Kill app diff feature flag

### PR DESCRIPTION
**Pull request by orangejenny**
Moved comparison options out of main deploy page, into source files page. Moved source files link out of main deploy page, into deploy modal - it's superuser-permissioned, as it was before https://github.com/dimagi/commcare-hq/pull/8299.

Before:
screen shot 2017-02-03 at 6 20 58 pm

and

screen shot 2017-02-03 at 6 21 36 pm

After:
screen shot 2017-02-03 at 6 19 38 pm

and

screen shot 2017-02-03 at 6 19 22 pm

@mkangia / @NoahCarnahan

### CODE REVIEW

**Comment by snopoke** [comment]
:+1: kill those feature flags. Didn't review closely but looks good

File: corehq/apps/app_manager/templates/app_manager/v1/source_files.html [hunk 1]
```diff
@@ -57,9 +57,15 @@
     {% compress js %}
         <script src="{% static 'hqwebapp/js/urllib.js' %}"></script>
         <script src="{% static 'hqwebapp/js/initial_page_data.js' %}"></script>
+        <script src="{% static 'Caret.js/dist/jquery.caret.js' %}"></script>
+        <script src="{% static 'At.js/dist/js/jquery.atwho.min.js' %}"></script>
```
Comment (NoahCarnahan) [inline corehq/apps/app_manager/templates/app_manager/v1/source_files.html]: Should we now prefer atwho to select2?

**Comment by NoahCarnahan** [comment]
Looks good thanks Jenny

File: corehq/apps/app_manager/templates/app_manager/v1/source_files.html (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v1/source_files.html]: No strong preference between the two. I used atwho here out of laziness (I'm more familiar with it that with select2)...but looked at it again today, and select2 makes the code a lot shorter. Will update.

Comment (NoahCarnahan) [inline corehq/apps/app_manager/templates/app_manager/v1/source_files.html]: Cool yeah I was just curious because I haven't used atwho before

**Pull request merged**